src/containers/StudyEdit/style.ts

```diff
@@ -14,14 +14,26 @@ export const StudyEditHeading = styled.div`
   gap: 2rem;
 `;
 
+export const InputWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+`;
+
+export const ErrorMessage = styled.div`
+  color: #ef4444;
```

Comment:
p4;
Muiprovider에 있는 것을 사용해도 좋아보입니다.
큰 문제는 아닙니다 :) 
```suggestion
  color: ${({theme}) => theme.palette.error}
```

Reply:
감사합니다👍 반영했습니다. 6343714316d6462ca463c1991a2956f7c724ebdb